### ISSUE TITLE
windows 10 install dapr 1.7.3 failed 

### ISSUE THREAD

**Issue by yaolc** [dapr/cli#996]
C:\WINDOWS\system32>dapr init --from-dir D:\daprbundle
Making the jump to hyperspace...
Local bundle installation using --from-dir flag is currently a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.
Installing runtime version 1.7.3
Extracting binaries and setting up components...
error moving dashboard binary to path: ʹ�á�3�����������á�SetEnvironmentVariable��ʱ�����쳣:���� ��ִ��δ����Ȩ�Ĳ�������
����λ�� ��:1 �ַ�: 1
+ [System.Environment]::SetEnvironmentVariable('Path',[System.Environme ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : UnauthorizedAccessException

C:\WINDOWS\system32>Error processing tar file(exit status 1): unexpected EOF

**Comment by shivamkm07** [comment]
@yaolc I tested with installer-bundle 1.7.3 on my windows machine and it worked well for me:

```
PS C:\Users\shivamkumar\demo\daprbundle_windows_amd64\daprbundle> .\dapr.exe init --from-dir .
Making the jump to hyperspace...
Local bundle installation using --from-dir flag is currently a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.
Installing runtime version 1.7.3
Extracting binaries and setting up components...
Loaded image: daprio/dapr:1.7.3
Extracted binaries and completed components set up.
daprd binary has been installed to C:\Users\shivamkumar\.dapr\bin.
dapr_placement container is running.
Use `docker ps` to check running containers.
Success! Dapr is up and running. To get started, go here: https://aka.ms/dapr-getting-started

PS C:\Users\shivamkumar\demo\daprbundle_windows_amd64\daprbundle> ls C:\Users\shivamkumar\.dapr\bin\


    Directory: C:\Users\shivamkumar\.dapr\bin


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        06-06-2022     14:38                web
-a----        06-06-2022     14:38      112238080 daprd.exe
-a----        06-06-2022     14:38       31488000 dashboard.exe


PS C:\Users\shivamkumar\demo\daprbundle_windows_amd64\daprbundle>
```

The error log reports issue in moving dashboard binary, which possibly might be because you could have dapr dashboard running in background and so windows would report error if you try to overwrite the dashboard binary. Can you please check if it's the case?

**Comment by yaolc** [comment]
@[shivamkm07](https://github.com/shivamkm07)
> The error log reports issue in moving dashboard binary, which possibly might be because you could have dapr dashboard running in background and so windows would report error if you try to overwrite the dashboard binary. Can you please check if it's the case?

this is the first time I installed dapr ,i try a lot of times,but failed.I was followed the step of daprbundle ,but the error still existed.
So I ask if any setting in my system didnot set.

PS C:\daprbundle> .\dapr.exe init --from-dir .
Making the jump to hyperspace...
Local bundle installation using --from-dir flag is currently a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.
Installing runtime version 1.7.0
Extracting binaries and setting up components...
.\dapr.exe : error moving dashboard binary to path: 使用“3”个参数调用“SetEnvironmentVariable”时发生异常:“尝试执行未经授权的操作。”
所在位置 行:1 字符: 1
+ .\dapr.exe init --from-dir .
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (error moving da...:“尝试执行未经授权的操作。”:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
所在位置 行:1 字符: 1
+ [System.Environment]::SetEnvironmentVariable('Path',[System.Environme ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : UnauthorizedAccessException
 
Error processing tar file(exit status 1): unexpected EOF

you can see that when dapr want to SetEnvironmentVariable,The system refused  because of the  Unauthorized

**Comment by yaolc** [comment]
I was fixed this problem by uninstall Antivirus software .Thanks
